tasks/security_agent.py

```diff
@@ -172,6 +172,61 @@ def functional_tests(
     ctx.run(cmd.format(**args), env=env)
 
 
+@task
+def stress_tests(
+    ctx,
+    race=False,
+    verbose=False,
+    go_version=None,
+    arch="x64",
+    major_version='7',
+    pattern='',
+    bench_pattern='',
+    output='',
+    build_tags='',
+    bundle_ebpf=True,
+    fail_fast=False,
+    keep_profile=False,
+    report_file='',
+    diff_base='',
```

Comment:
Do we expect all of these flags to be used? When would they be used? I feel like we are unnecessarily duplicating the `go test` CLI interface. We could just have a few key options on the invoke task and a `testflags` option to directly pass arbitrary flags to `go test`.

Reply:
Awesome, that looks good to me! Marking as solved